build/gitversion.yml

```diff
@@ -0,0 +1,35 @@
+assembly-versioning-scheme: MajorMinorPatch
+mode: ContinuousDeployment
+next-version: 0.4.0

```

Comment:
Not sure we had a discussion on version numbers, not sure what's the procedure about it